### ISSUE TITLE
Fails to build out-of-box on a new project

### ISSUE THREAD

**Issue by EricR86** [feed-rs/feed-rs#89]
On my machine if a create a new project with `cargo new` and add `feed-rs="0.4"` as my only dependency I get the following error:
```
   Compiling feed-rs v0.4.1
error[E0603]: module `export` is private
   --> C:\.cargo\registry\src\github.com-1ecc6299db9ec823\feed-rs-0.4.1\src\xml\mod.rs:10:12
    |
10  | use serde::export::Formatter;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> C:\.cargo\registry\src\github.com-1ecc6299db9ec823\serde-1.0.120\src\lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0603`.
error: could not compile `feed-rs`
```
Maybe this was due an update with serde?

**Comment by markpritchard** [comment]
Hi @EricR86 , glad to hear you are looking into the library! 

This was fixed recently in #88 , but I haven't cut a release yet. You could try using a git sha for the dependency, and then update it once the new release is done (in the next few days).

**Comment by jangernert** [comment]
I hate to be "that guy" again. But is it possible to get a new point release soonish? I would like to release a new version of [NewsFlash](https://flathub.org/apps/details/com.gitlab.newsflash) and would have to downgrade serde to make it possible otherwise.

**Comment by markpritchard** [comment]
Its OK @jangernert , squeaky wheel and all that :) 

I was just finalising the MediaRSS support and merging it in. Will then cut another release.

**Comment by markpritchard** [comment]
Fixed in #91

**Comment by jangernert** [comment]
@markpritchard I didn't know you were so close to finishing the MediaRSS support. In that case I will put that feature to use and release once that is hooked up :)
